src/components/Form/DocumentSelection.jsx

```diff
@@ -20,14 +20,17 @@ import { docTypes } from '../../utils';
 const DocumentSelection = ({ htmlForAndIdProp, handleDocType, docType }) => (
   <Box sx={{ marginTop: '20px' }}>
     <FormControl sx={{ width: 220 }}>
-      <InputLabel id={`${htmlForAndIdProp}-label`}>Select Document Type</InputLabel>
+      <InputLabel id={`${htmlForAndIdProp}-label`} required>
```

Comment:
The required in Line 23 and Line 33 should be on Line 22 instead for `<FormControl>` as mentioned previously (https://github.com/codeforpdx/PASS/pull/253#discussion_r1231631977).

https://github.com/codeforpdx/PASS/assets/14917816/70e8d1c8-8a18-4b0e-a0b2-29b3f26844b6